.github/workflows/test.yml

```
@@ -19,6 +20,11 @@ jobs:
             gemfile: Gemfile-rails.5.2.x
           - ruby: 3.0
             gemfile: Gemfile-rails.5.2.x
+          - ruby: 2.5
+            gemfile: Gemfile-rails.7.0.x
+          - ruby: 2.6
+            gemfile: Gemfile-rails.7.0.x
```

Looks like these need to point to `Gemfile` now:

```suggestion
          - ruby: 2.5
            gemfile: Gemfile
          - ruby: 2.6
            gemfile: Gemfile
```